src/pages/home/home.tsx

```diff
@@ -1,4 +1,5 @@
 import FirstComment from '~components/FirstComment/FirstComment';
+import Icon from '~components/Icon/IconFont';
```

Comment:
Mentenha a mesma nomeclatura da pasta e do component file

```suggestion
import Icon from '~components/Icon/Icon';
```

Reply:
Feito!